src/foundReducer.ts

```diff
@@ -21,12 +21,12 @@ const foundReducer = (
     case ActionTypes.RESOLVE_MATCH:
       // It doesn't make sense to resolve a match if there wasn't already an
       // unresolved match.
-      return state
-        ? {
-            match: state.match,
-            resolvedMatch: payload,
-          }
-        : null;
+      return (
+        state && {
```

Comment:
Can state be some non null value here ?

Reply:
`state` should never be null here afair, but since this is technically a functional change i reverted it too